epytope/EpitopePrediction/External.py

```diff
@@ -6466,14 +6502,20 @@ def parse_external_result(self, file):
         :return: A dictionary containing the prediction results
         :rtype: dict
         """
-        result = defaultdict(defaultdict)
+        scores = defaultdict(defaultdict)
+        alleles = []
         with open(file, "r") as f:
             for row in f:
                 if row[0] in ["#", "-"] or row.strip() == "" or "pos" in row:
                     continue
                 else:
-                    s = row.split()
-                    result[s[1].replace("*", "")][s[2]] = float(s[4])
+                    allele, pep, score = Allele(row.split()[HLAIndex.PICKPOCKET_1_1]), Peptide(row.split()[PeptideIndex.PICKPOCKET_1_1]), float(row.split()[ScoreIndex.PICKPOCKET_1_1])
+                    if allele not in alleles:
+                        alleles.append(allele)
+                    scores[allele][pep] = score
+
+            result = {allele: {"Score": list(scores.values())[j]} for j, allele in enumerate(alleles)}
```

Comment:
I won't comment on future occurrences, let's just search for all occurrences for the next commit

Reply:
Adressed in #43